tests/components/Microsoft.DotNet.UpgradeAssistant.Analysis.Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "profiles": {
+    "Microsoft.DotNet.UpgradeAssistant.Analysis.Tests": {
+      "commandName": "Project"
+    }
+  }
+}
```

Comment:
Why is launchSetting.json needed for tests?

Reply:
It shouldn't be, although I think you can launch it to run the tests.